cibyl/exceptions/config.py

```diff
@@ -103,7 +93,18 @@ class MissingSourceKey(CibylException):
     def __init__(self, source_type, key):
         colored_key = Colors.blue(key)
         self.message = f"""The following key in "{source_type}" source type \
-is missing and required for the source to become operational: {colored_key}."""
+is missing and required for the source to become operational: {colored_key}"""
+
+        super().__init__(self.message)
+
+
+class MissingSystemKey(CibylException):
+    """System configuration is incomplete and missing a key."""
+
+    def __init__(self, system_name, key):
+        colored_key = Colors.blue(key)
+        self.message = f"""The following key in "{system_name}" system \
+is missing and required for the system to become operational: {colored_key}"""
```

Comment:
Minor thing, but I find this lack of indentation... disturbing.
Or at least it frightens me, but it is already present in other places, so I can pretend like its not here.

Reply:
I'm open for suggestions :)